packages/shared/src/components/modals/award/BuyCoresModal.tsx

```diff
@@ -220,7 +276,17 @@ export const BuyCoresProcessing = ({ ...props }: ModalProps): ReactElement => {
       }
 
       if (transactionStatus === UserTransactionStatus.Error) {
-        // TODO show error message
+        setActiveStep({
+          step: 'PROCESSING_ERROR',
+        });
+        setProcessingError({
+          title: 'There was an issue processing your transaction.',
+          description:
+            'Please check your payment method and try again. If the issue persists, please contact support.',
```

Comment:
I think in this case we should say to contact support. Because most probably issue was not due to payment.

Reply:
It says contact support?

Reply:
Yeah but the check payment method is worthless because most likely its not about that.